my-sms/app/views/institutes/edit.html.haml

```diff
@@ -1,2 +1,5 @@
-%h1 Institutes#edit
-%p Find me in app/views/institutes/edit.html.haml
+- model_class = Institute
```

Comment:
Do you really need this variable? Am just worried about variables polluting the view in case they're not essential.

Reply:
I was just following the same pattern that scaffolding set up on the original Student edit page. It does make copy/pasting code to new models efficient.

Reply:
Copy-pasting oriented programming.